hazelcast/src/main/java/com/hazelcast/internal/cluster/MemberInfo.java

```diff
@@ -39,7 +39,7 @@
 import static com.hazelcast.internal.serialization.impl.SerializationUtil.writeMap;
 import static com.hazelcast.internal.util.MapUtil.createHashMap;
 
-public class MemberInfo implements IdentifiedDataSerializable, Versioned {
+public class MemberInfo implements IdentifiedDataSerializable {
```

Comment:
If you remove `Versioned`, a 5.2 member reading a `MemberInfo` sent from a 5.3 member will consider its version to be `UNKNOWN` -> `isGreaterOrEqual` will return `false` -> it won't read the UUID even though it is sent from 5.3 member.
It will be safe to remove `Versioned` interface in 5.4.

Reply:
Done

Reply:
nit: @krzysztofslusarski maybe you want to add that to the class name